src/loginServices.js

```diff
@@ -31,6 +31,7 @@ export const supportedLocales = [
   'ar',     // arabic
   'zh-CN',  // chinese, china
   'zh-TW',  // chinese, taiwan
+  'cs_CZ',  // czech, czechia
```

Comment:
@zburke - Is it okay that Czech uses an underscore as a separator (as it should be) or at this point should it be a dash (as it has historically been)?

Reply:
Whoops, that's a bug. Sharp eyes, thank you! [The dash is correct (RFC-5646)](https://datatracker.ietf.org/doc/html/rfc5646#section-2.1). I don't know the our underscore convention came from, if it's a peculiarity of Lokalise or something we did by accident.

Reply:
Yes, unfortunately that decision is lost to the mists of time.  I have a vague memory that there was a reason why we did it this way, but I've lost all hope of trying to reconstruct that reasoning.  

Thanks, Zak.